projects/plugins/wpcomsh/tests/WpcomshTest.php

```diff
@@ -60,4 +60,20 @@ public function test_wpcomsh_make_content_clickable() {
 
 		$this->assertEquals( $expected_output, wpcomsh_make_content_clickable( $original_content ) );
 	}
+
+	/**
+	 * Tests if Jetpack Boost plugin is active, to test the integreation setup.
+	 *
+	 * This is for the `jp docker phpunit-integration` command to verify it works.
+	 *
+	 * @return void
+	 */
+	public function test_is_jetpack_boost_active() {
+		$plugins = getenv( 'JP_MONO_INTEGRATION_PLUGINS' );
+		if ( $plugins && strpos( $plugins, 'boost' ) !== false ) {
+			$this->assertTrue( is_plugin_active( 'boost/jetpack-boost.php' ) );
```

Comment:
I note this will probably only pass within the monorepo environment, because everywhere else it'd be "jetpack-boost/jetpack-boost.php".

That's not necessarily a problem, since that's probably also about the only environment we support running these tests in, but it may be worth noting.